lib/chef-dk/helpers.rb

```diff
@@ -48,13 +48,22 @@ def stderr
     end
 
     #
-    # Locates the omnibus directories
+    # Returns true if this DK installation is identified as
+    # an omnibus package
     #
-
     def omnibus_install?
       File.exist?(omnibus_chefdk_location)
     end
 
+    #
+    # Returns true if this DK/WS installation is identified as a
+    # habitat package.  Release on the environment variable
+    # 'VIA_HABITAT', which gets set in the wrapper scripts
+    # generated in plan.sh
+    def habitat_install?
```

Comment:
Cool. I wonder about having both habitat and omnibus install functions. seems like they are mutually exclusive but have different logic. Let's leave it. Just stood out to me.

Reply:
I had originally gone with "!omnibus", but that resulted in an assumption that !omnibus meant 'hab == true', which wasn't always correct. 

Given that this is going to live on for some time, I wouldn't object to factoring out some kind of `InstallPlatform` to capture installation-source decisions and behaviors in our next pass